vms/organization/templates/organization/list.html

```diff
@@ -17,6 +50,7 @@
         </thead>
         <tbody>
             {% for organization in organization_list %}
+                 {% if organization.approved_status is 1 %}
```

Comment:
Any reason for not doing a bool check?

Reply:
Actually, she has divided the organizations into three categories namely, approved, pending and rejected and the corresponding integer values are 1, 0 and 2 respectively.

Reply:
oh ok then its fine